terraform/worker_uk.tf

```diff
@@ -70,6 +70,10 @@ module "worker_uk" {
       {
         name  = "ENVIRONMENT"
         value = var.environment
+      },
+      {
```

Comment:
I don't think you need this chipolata since its configured in the common worker vars merged in from the locals